src/pages/payment-methods/payment-methods-translation.js

```diff
@@ -0,0 +1,398 @@
+import React from 'react'
+import { localize } from 'components/localization'
```

Comment:
@sean-binary  this is the file generated by the payment_method script to get handled by translation extractor script